src/app-layer-htp.c

```diff
@@ -99,113 +99,74 @@ static uint64_t htp_state_memuse = 0;
 static uint64_t htp_state_memcnt = 0;
 #endif
 
-SCEnumCharMap http_decoder_event_table[ ] = {
-    { "UNKNOWN_ERROR",
-        HTTP_DECODER_EVENT_UNKNOWN_ERROR},
-    { "GZIP_DECOMPRESSION_FAILED",
-        HTTP_DECODER_EVENT_GZIP_DECOMPRESSION_FAILED},
-    { "REQUEST_FIELD_MISSING_COLON",
-        HTTP_DECODER_EVENT_REQUEST_FIELD_MISSING_COLON},
-    { "RESPONSE_FIELD_MISSING_COLON",
-        HTTP_DECODER_EVENT_RESPONSE_FIELD_MISSING_COLON},
-    { "INVALID_REQUEST_CHUNK_LEN",
-        HTTP_DECODER_EVENT_INVALID_REQUEST_CHUNK_LEN},
-    { "INVALID_RESPONSE_CHUNK_LEN",
-        HTTP_DECODER_EVENT_INVALID_RESPONSE_CHUNK_LEN},
+SCEnumCharMap http_decoder_event_table[] = {
```

Comment:
please put the reformat into its own commit

Reply:
ok